### DISCUSSION TITLE
Processor key in "Provenance"

### DISCUSSION THREAD

**Discussion by bastienird** [eblondel/geoflow#313] [Q&A] [answered]
We used to have processor key in "Provenance" columns in our workflow, however, it does not seem to be allowed anymore ("Error : Key 'processor' is invalid, allowed key values are [statement,process]"). 

I tried as it is written in the doc to add as many "processor" as "process" and I am getting the same error. Is processor being removed, and if yes, can it be changed in the doc ?

Thanks, 

Bastien

**Comment by eblondel** [comment, marked as answer]
Please rely on https://github.com/r-geoflow/geoflow/issues/303 let me know if you have issues

**Reply by eblondel** [reply]
@BastienIRD Long story short, `processor` is not anymore accepted. You can define a `statement` and a set of `process`. The `processor` can be defined in the `Creator` (contacts) column, eventually appended with a prefix to refer to some `process` in particular. In this way we can handle eventual multiple `processor` contacts by each `process` which is allowed in the ISO 19115 standard; in more flexible way, and we keep homogeneity of metadata contact, meaning that contacts whatever their role is, should be managed in the `Creator` column.
